plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1004,6 +1004,19 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       duration = animationTime
     }
 
+    zoomAnimator.addListener(object : AnimatorListenerAdapter() {
+      override fun onAnimationStart(animation: Animator?) {
+        super.onAnimationStart(animation)
+        // notify scale gesture started when zoom animation starts.
+        notifyOnScaleListeners(gesturesManager.standardScaleGestureDetector)
```

Comment:
Should it be `notifyOnScaleBeginListeners` here?

Reply:
I used `notifyOnScaleBeginListeners` when `handleZoomAnimation` is called, that way we notify that animation is starting. Although we could use that here, and use `notifyOnScaleListener` in `onAnimationRepeat` (that will update gesture in progress continuously). I have checked with other gestures and we only notify the gesture in progress when the animation is about to start, rather than in `onAnimationRepeat`, probably cause we don't want to flood the `gesture in progress` calls. so that's why I have used the same callback flow here too. 

`OnScaleBegin` called when `handleZoomAnimation` is called.
`OnScale` when Animation is about to start.
`onScaleEnd` when Animation will be canceled / end